### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by SmythConor**
## Ready for merging
- Need to get a name for the linear regression function

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -13,6 +16,14 @@
  */
 public class Main {
 	public static void main(String[] args) {
+		/* Simple classification and prediction described in Report */
+		simpleClassificationAndPrediction();
+
+		/* Linear Regression attempt described in Report */
+		whatWillICallThis();
```
Comment (SmythConor) [inline src/main/java/Main.java]: **Need a name for this!** reply with a name xo

File: src/main/java/Main.java [hunk 2]
```diff
@@ -108,4 +119,137 @@ public static void main(String[] args) {
 
 		System.out.println("Player to win: " + winnerName);
 	}
+
+	public static void whatWillICallThis() {
```
Comment (SmythConor) [inline src/main/java/Main.java]: **NAMEUUHHHHH**

File: src/main/java/Main.java (hunk 1 above)
Comment (aoflynn) [inline src/main/java/Main.java]: call it ExponentialRegression

File: src/main/java/Main.java [hunk 3]
```diff
@@ -108,4 +119,137 @@ public static void main(String[] args) {
...
+		int size = DataInfo.getTopPlayers();
```
Comment (SmythConor) [inline src/main/java/Main.java:124]: Added this so that the sizes aren't hard coded later on, allows for easier changes

**Pull request merged**